### DISCUSSION TITLE
Quick styling and branding of superset app

### DISCUSSION THREAD

**Discussion by mararual** [apache/superset#12823] [Q&A / Help]
hey everyone. Question: How can I change the superset logo, navbar colors and font maybe? It’s for a PoC that I want to make it look more branded.

**Comment by nytai** [comment]
You can change the logo via the config setting `APP_ICON`
https://github.com/apache/superset/blob/ef724069f31508a002afa4a5ae50fcc3ea119cd5/superset/config.py#L197

As for changing the menu colors, that's a little harder.
The theme variables are held in this package 
https://github.com/apache-superset/superset-ui/blob/87f16cc37d15732e9ac80f3135a2540d7d93fbbc/packages/superset-ui-core/src/style/index.ts#L36

You can override some of them by looking for lines that look like this
https://github.com/apache/superset/blob/ef724069f31508a002afa4a5ae50fcc3ea119cd5/superset-frontend/src/views/App.tsx#L71

Then adding your overrides, something like 
```js
<ThemeProvider theme={{…supersetTheme, …myOverrides}}>
```

**Reply by mararual** [reply]
@nytai  thanks! I can change the app_icon easily. Regarding the other styles and the theme provider, it would require to modify the source code then and buuild from there. Right now I’ using the docker image as base and modified with plugins.

or maybe I can create my own style files and overwrite them through docker when building the image?

**Reply by nytai** [reply]
Yup, depending on your setup forking and maintain some diverging commits might work otherwise handling all of this a build time could also work.

**Reply by nytai** [reply]
We haven't looked at custom theming much, so unfortunately we don't have an easy way for users to override the styles. I think eventually we should support specifying a path to overrides via config.

**Reply by Lighttree** [reply]
Hi guys. Are there any changes regarding this? Modifying the source code of the client app is still the way for "theming" Superset? Just wondering about "update lock" in this case.
